app/src/test/java/org/gnucash/android/test/unit/export/QifExporterTest.java

```diff
@@ -144,4 +149,46 @@ public void multiCurrencyTransactions_shouldResultInMultipleQifFiles(){
         assertThat(file.length()).isGreaterThan(0L);
     }
 
+    //@Test
+    public void description_and_memo_field_test() {
```

Comment:
Yes, I also disagree with some conventions, but not following them just makes the code stand out unnecessarily.

The tests combining the two styles is also a convention :-) I can't find the reference now, but more or less it follows the format: `someAction_whatDoYouExpectFromIt()`.